CHANGES.md

```diff
@@ -1,6 +1,362 @@
+  (see https://chapel-lang.org/docs/1.29/modules/standard/IO.html#IO.unicodeSupported)
+* deprecated `string`/`bytes` casts to `regex` in favor of `Regex.compile()`  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/Regex.html#Regex.compile)
+* deprecated `copyMode()`, `getMode()`, and `chmod()` from 'FileSystem'
+* deprecated `createVersion()` in favor of `new versionValue()`  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/Version.html#Version.createVersion)
+* removed `"version "` prefix when casting a `sourceVersion` to string  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/Version.html#Version.sourceVersion)
+* deprecated the 'SysBasic' module including `EEOF`, `ESHORT`, `EFORMAT`
+* marked 'GPU'/'GPUDiagnostics' modules as unstable (due to still being new))
+* marked `iostringstyle`, `iostringformat` as unstable in the 'IO' module
+* removed the deprecated 'SysError' module and some related deprecated symbols
+* removed the deprecated 'DateTime' module
+* removed deprecated support for `start`/`end` arguments from 'IO' routines
+* removed `tryGetPath()`, `unlink()`, and `getchunk()` from 'IO'
+* removed deprecated `iohint` type from the 'IO' module
+* removed deprecated formals in channel methods `.search()`, `.match()`, etc.
+* removed support for deprecated formals in `regex.compile()`
+* removed deprecated class `BadRegexpError`
+* removed deprecated fields `.offset`, `.size` in `regexMatch`
+* removed deprecated `.indexOf()`, `match()` methods on `list`
+* removed deprecated `map` operators
+
+Standard Library Modules
+------------------------
+* added `readAll()` methods to read the entire contents of a `fileReader`  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/IO.html#IO.channel.readAll)
+* added six new overloads of `read`|`writeBinary()` for a variety of types  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/IO.html#IO.channel.readBinary  
+   and https://chapel-lang.org/docs/1.29/modules/standard/IO.html#IO.fileWriter.writeBinary)
+* modified `file.path` to exclusively return absolute, not relative, paths  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/IO.html#IO.file.path)
+* removed unused `bool` return values from various `write*()` routines
+* added new `.reset()` and `.restart()` methods to the `stopwatch` type  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/Time.html#Time.stopwatch.reset  
+   and https://chapel-lang.org/docs/1.29/modules/standard/Time.html#Time.stopwatch.restart)
+* added a new `version` type that can be constructed/modified at run-time  
+  (see https://chapel-lang.org/docs/1.29/modules/standard/Version.html#Version.version)
+* 'CommDiagnostics' no longer counts communications for tracking remote tasks
+
+Package Modules
+---------------
+* optimized the flushing performed in the 'CopyAggregation' module
+* improved support for matrices made of het. array literals in 'LinearAlgebra'
```

Comment:
`het.` caught my eyes thinking it was a two sentence entry, maybe:

```
* improved 'LinearAlgebra' support for matrices of heterogeneous array literals
```

Reply:
Man, who contributed that text?!?